packages/core/src/dropdown/Dropdown.tsx

```diff
@@ -98,7 +99,15 @@ export type DropdownProps<Item = string> = {
   ListControlProps<Item>;
 
 function ExpandIcon({ open }: { open: boolean }) {
-  return open ? <ChevronUpIcon aria-hidden /> : <ChevronDownIcon aria-hidden />;
+  const {
+    SemanticChevronUpIcon = ChevronUpIcon,
+    SemanticChevronDownIcon = ChevronDownIcon,
+  } = useSemanticIcon();
+  return open ? (
```

Comment:
Do we need to prefix useSemanticIcon, could it be simpler, useIcon instead?. You typically only start to prefix or suffix when there is more than one of something.